### PR TITLE
Update UI aesthetics

### PR DESCRIPTION
**Pull request by declanleeler**
Let's:
* improve color theme
* improve aesthetics of the results list
* change the title
* reposition command box and response display

### CODE REVIEW

File: src/main/resources/view/OrderListPanel.fxml [hunk 1]
```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (darshan152) [inline src/main/resources/view/OrderListPanel.fxml:6]: Should this be changed back to 8? PersonListPanel.fxml uses `http://javafx.com/javafx/8` still. Should be standardized throughout our fxml files.

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -9,46 +9,43 @@
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.stage.Stage?>
 
-<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+<fx:root minHeight="600" minWidth="450" onCloseRequest="#handleExit" title="Address App" type="javafx.stage.Stage"
+         xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (darshan152) [inline src/main/resources/view/MainWindow.fxml:15]: Javafx version should be standardized

**Review by darshan152 — CHANGES_REQUESTED** [review]
JavaFX version should be changed to a single version in all files, the change to ver17 is probably due to using scene builder, can just manually change back the numbers to 8 and it should work fine

File: src/main/resources/view/OrderListPanel.fxml (hunk 1 above)
Comment (declanleeler) [reply src/main/resources/view/OrderListPanel.fxml:6]: Good catch, Ill update this

**Review by darshan152 — APPROVED** [review]
LGTM

**Pull request merged**